tools/code-owners-parser/CodeOwnersParser/CodeOwnerEntry.cs

```diff
@@ -130,5 +133,32 @@ private string ParsePath(string line)
             // remove the path from the string.
             return line.Substring(ownerStartPosition);
         }
+
+        public void FilterOutTeamAlias()
+        {
+            client.BaseAddress = new Uri("https://api.github.com/");
+            client.DefaultRequestHeaders.UserAgent.Add(new System.Net.Http.Headers.ProductInfoHeaderValue("CodeOwnerRetriever", "1.0"));
+            client.DefaultRequestHeaders.Accept.Add(new System.Net.Http.Headers.MediaTypeWithQualityHeaderValue("application/json"));
+            Owners.RemoveAll(r => IsTeamAlias(r));
+        }
+
+        private static bool IsTeamAlias(string alias)
+        {
+            var userUriStub = $"users/{alias}";
```

Comment:
Given we are filtering to users as opposed to directly checking teams we should change the naming to IsUserAlias